src/app/components/core/configuration-manager/configuration-group/configuration-group.component.ts

```diff
@@ -147,7 +155,7 @@ export class ConfigurationGroupComponent implements AfterViewInit {
 
   buildGroupOfItems(configItems) {
     configItems?.forEach(config => {
-      if(config.readonly != 'true'){
```

Comment:
What if readonly does not exist?

Reply:
This line is not removed from here, there is only indentation fix (by editor), please check.